scss/_functions.scss

```diff
@@ -53,13 +56,28 @@
   $r: red($color);
   $g: green($color);
   $b: blue($color);
+  // get the relative lum for each color
+  $luminositecouleur: relativelum($r / 255, $g / 255, $b / 255);
```

Comment:
small request: as the official language for our files is english, could you update the variables to `$luminositycolor` and `$luminositywhite` ? merci :)